src/Text/Pandoc/Readers/Org.hs

```diff
@@ -472,7 +483,7 @@ rawBlockContent (indent, blockType) = try $
   unlines . map commaEscaped <$> manyTill indentedLine blockEnder
  where
    indentedLine = try $ ("" <$ blankline) <|> (indentWith indent *> anyLine)
-   blockEnder = try $ indentWith indent *> stringAnyCase ("#+end_" <> blockType)
+   blockEnder = try $ skipSpaces *> stringAnyCase ("#+end_" <> blockType)
```

Comment:
actually fwiw I think that Emacs' behavior is closer to `skipSpaces`. For instance,

```org
#+BEGIN_SRC org
  #+BEGIN_SRC haskell
  putStrLn "HELLO"
  #+END_SRC
#+END_SRC
```

Gets parsed as follows:

https://gist.github.com/9e1a533d377153cc5604374b53b2ea9a

(i.e., the first `END_SRC` does get interpreted as the end.)